packages/airnode-protocol/contracts/authorizers/RequesterAuthorizer.sol

```diff
@@ -153,14 +157,15 @@ abstract contract RequesterAuthorizer is Whitelist, IRequesterAuthorizer {
     /// @param sponsor Sponsor address
     /// @param requester Requester address
     /// @return Authorization status of the request
-    function isAuthorized(
+    function isAuthorizedV0(
```

Comment:
Compatible with AuthorizationUtilsV0. The node never calls this directly so it doesn't need to be updated.